config/environments/development.rb

```diff
@@ -58,10 +56,6 @@
   # Annotate rendered view with file names.
   # config.action_view.annotate_rendered_view_with_filenames = true
 
-  # Use an evented file watcher to asynchronously detect changes in source code,
-  # routes, locales, etc. This feature depends on the listen gem.
-  config.file_watcher = ActiveSupport::EventedFileUpdateChecker
```

Comment:
[TODO]
I'm not sure which one we should use from v7.0.0. I will search for it.
- `ActiveSupport::EventedFileUpdateChecker`(depends on gem `listen`)
- `ActiveSupport::FileUpdateChecker`

Reply:
In default setting, `ActiveSupport::EventedFileUpdateChecker` is removed.
So we will accept this changes and remove `gem listen` in the following PR.
- https://github.com/rails/rails/commit/e34300a921a910c60cd4064038b0817fe0e0ab18